File: ARRAYS/ArrayCreation/arr.py
def insert(arr,element):
    arr.append(element)
    return arr

if __name__ == "__main__":
     LA=[0,0,0]
     x=0

for x in range(len(LA)):
    print("LA[",x,"]=",LA[x])

for x in range(len(LA)):
    LA.append(x)
    LA[x]= x+1

for x in range(len(LA)):
    print("LA[",x,"]=",LA[x])

File: ARRAYS/ArrayUpdate/arr.java
package ARRAYS.ArrayUpdate;

public class arr {
    public static void main(String []args) {
        int LA[] = new int[3];
        int n = LA.length;
        System.out.println("Array Before Update:");
        for(int i = 0; i < n; i++) {
            LA[i] = i + 3;
            System.out.println("LA[" + i + "] = " + LA[i]);
        }
        //Updating an element in an array
        LA[1] = 10;
        System.out.println("Array After Update:");
        for(int i = 0; i < n; i++) {
            System.out.println("LA[" + i + "] = " + LA[i]);
        }
    }
}
